_includes/geek-details.html

```diff
@@ -0,0 +1,64 @@
+<div class="geek-details-wrapper">
+    <div class="geek-details">
+        {% assign details = include.details | split: ',' %}
+        {% for detail in details %}
+            {% assign kv = detail | split: ':' %}
+            <div class="geek-detail">
+                {{ kv[0] }}
+                <div class="geek-detail-value" id="geekValue{{ kv[0] }}">
+                    {{ kv[1] }}
+                </div>
+            </div> 
+            <div class="modal fade" id="geekModal{{ kv[0] }}"  role="dialog">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                        <h4 class="modal-title">{{ kv[1] }} {{ kv[0] }}</h4>
+                        </div>
+                        <div class="modal-body">
+                        {% capture geek-details-file %}geek-details/{{kv[0] | downcase}}-{{kv[1] | downcase | replace: " ", "-"}}.md{% endcapture %}
+                        {% capture geek-details-md %}{% include {{geek-details-file}} %}{% endcapture %}
+                        {{ geek-details-md | markdownify }}
+                        </div>
+                        <div class="modal-footer">
+                        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                        </div>
+                    </div>
+                </div>
+            </div>    
+            <script>
+                $(document).ready(function(){
+                $("#geekValue{{ kv[0] }}").click(function(){
+                    $("#geekModal{{ kv[0] }}").modal();
+                });
+                });
+            </script>    
+        {% endfor %}
+    </div>
+    <button class="geek-details-info" id="geekDetailsInfo">?</button>
```

Comment:
When I click the question mark, I'm getting details about "Kubernetes Datastore". Is that the intent? I'm on chrome

Reply:
Oh wait, I'm getting general info, but the title is "Kubernetes Datastore".

Reply:
Good catch - will fix.

Reply:
Fixed